kubernetes/resource_kubernetes_daemon_set_v1_test.go

```diff
@@ -74,6 +74,7 @@ func TestAccKubernetesDaemonSetV1_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "spec.0.template.0.spec.0.container.0.name", "tf-acc-test"),
 					resource.TestCheckResourceAttr(resourceName, "spec.0.strategy.0.type", "RollingUpdate"),
 					resource.TestCheckResourceAttr(resourceName, "spec.0.strategy.0.rolling_update.0.max_unavailable", "1"),
+          resource.TestCheckResourceAttr(resourceName, "spec.0.strategy.0.rolling_update.0.max_surge", "0"),
```

Comment:
You'll want to address some formatting issues found in this file. See: https://github.com/hashicorp/terraform-provider-kubernetes/actions/runs/11505331384/job/32028240254?pr=2609

Reply:
Yeah, I had just noticed that. Thanks!